odh-dashboard/apps/habana/habana-quickstart.yaml

```
@@ -0,0 +1,239 @@
+
+        def main():
+
+            epochs = 20
+            batch_size = 128
+            lr = 0.01
+            milestones = [10,15]
+            load_path = './data'
+            save_path = './checkpoints'
+
+            if(not os.path.exists(save_path)):
+                os.makedirs(save_path)
+            
+            # Target the Gaudi HPU device
+
+            device = torch.device("hpu")
+
+            
+            # Data
+            transform = transforms.Compose([
+                transforms.ToTensor(),
+            ])
+
+            trainset = torchvision.datasets.MNIST(root=load_path, train=True,
+                                                    download=True, transform=transform)
+            trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size,
+                                                    shuffle=True, num_workers=2)
+            testset = torchvision.datasets.MNIST(root=load_path, train=False,
+                                                download=True, transform=transform)
+            testloader = torch.utils.data.DataLoader(testset, batch_size=batch_size,
+                                                    shuffle=False, num_workers=2)
+
+            net = SimpleModel()
+            net.to(device)
+
+            criterion = nn.CrossEntropyLoss()
+            optimizer = optim.SGD(net.parameters(), lr=lr,
+                                momentum=0.9, weight_decay=5e-4)
+            scheduler = torch.optim.lr_scheduler.MultiStepLR(optimizer, milestones=milestones, gamma=0.1)
+
+            for epoch in range(1, epochs+1):
+                print("=====================================================================")
+                print("Epoch : {}".format(epoch))
+                train(net,criterion,optimizer,trainloader,device)
+                test(net,criterion,testloader,device)
+
+                torch.save(net.state_dict(), os.path.join(save_path,'epoch_{}.pth'.format(epoch)))
+
+                scheduler.step()
+
+        if __name__ == '__main__':
+            main()
+        ```
+      review:
+        failedTaskHelp: This task is not verified yet. Try the task again.
+      summary:
+        success: Continue 
+        failed: Try the steps again.
+    - title: Review the code 
+      description: |-
+        The example.py presents a basic PyTorch code example. The Habana-specific lines are explained below.
+        ### Line 10 - Import Habana Torch Library:
+        ```
+        import habana_frameworks.torch.core as htcore
+        ```
+        ### Line 104 - Target the Gaudi HPU device:
+        ```
+        device = torch.device("hpu")
+        ```
+        ### Lines 47, 52 - In Lazy mode, mark_step() must be added in all training scripts right after loss.backward() and optimizer.step().
+        ```
+        htcore.mark_step()
+        ```
+      review:
+        failedTaskHelp: This task is not verified yet. Try the task again.
```

Same here, try to add something like this:

```suggestion
        instructions: |-
          #### Verify the lines are present in the file:
          Do you have the lines in the file?
        failedTaskHelp: This task is not verified yet. Try the task again.
```